Inc/HALALMock/Services/Communication/FDCAN/FDCAN.hpp

```diff
@@ -58,18 +59,17 @@ class FDCAN{
      *        predefined instances should be used.
      *
      */
+    uint8_t Port_counter; //Puede ser inutil, por qie fdcam_number ya esta, pero bueno por si acaso
```

Comment:
```suggestion
    uint8_t port_counter;
```
no need for the comment, it is already being used